notebooks/Testgrid_flakiness_detection.ipynb

```diff
@@ -0,0 +1,2907 @@
+{
```

Comment:
<blockquote>flakiness score dictionary using Flip Flakiness with Optimal Distance method is: {(4, 7): 100.0, (31, 37): 85.71428571428571}</blockquote><p> </p><p>This output raises the question, what are we trying to identify here as one can't really directly compare these metrics? Are we trying to highlight tests that are themselves flaky based on their total recorded behavior, or individual test runs or regions of flakiness? How can you compare these metrics if the first two represent the entire set of runs and the third represent subsets?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/37/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='64'/>

Reply:
So, the main point over here, 
It is not a good idea to calculate a single flakiness score for each row. because sometimes the signal might be low due to large number of runs.


> This output raises the question, what are we trying to identify here as one can't really directly compare these metrics?
> How can you compare these metrics if the first two represent the entire set of runs and the third represent subsets?

Good point. I added the total detected failure by each method for comparison. 
Also, added detailed documentation.